docs/README.md

```diff
@@ -279,9 +279,11 @@ Therefore, in order to customize the `product-list` configuration, you can simpl
 
 ### `unit-price` props
 
-| Prop name              | Type      | Description                                                                                                                                                                                                                                     | Default value |
-| ---------------------- | --------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------- |
-| `textAlign`            | `string`    | Product unit prices position on the list.  | `left`    |
+| Prop name   | Type     | Description                               | Default value |
+| ----------- | -------- | ----------------------------------------- | ------------- |
+| `textAlign` | `string` | Product unit prices position on the list. | `left`        |
+| `unitPriceDisplay` | `string` | Defines whether the unit price will always appear (`always`) or only when the quantity of products is greater than one (`default`). | `default`        |
+| `displayUnitListPrice` | `string` | Defines whether the list-price will appear when it is different from the price (`showWhenDifferent`) or if it should never appear (`notShow`). | `left`        |
```

Comment:
```suggestion
| `unitPriceDisplay` | `enum` | Defines when the unit price should be displayed. Possible values are: `always` (unit price is always displayed) or `default` (unit price is only displayed when the number of products is greater than one). | `default`        |
| `displayUnitListPrice` | `enum` | Defines whether the product list price should be displayed or not. Possible values are: `showWhenDifferent` (list price is displayed when it is different from the regular price) or`notShow` (list price is never displayed). | `notShow`        |
```

Reply:
I'm not sure it `notShow` is the default value for `displayUnitListPrice` prop.

Reply:
@guerreirobeatriz suggestion applied. Actually the unit list price does not shows up, this pull request implements it.